### ISSUE TITLE
Batch highlight change/different

### ISSUE THREAD

**Issue by kiske666** [notepad-plus-plus/notepad-plus-plus#14244]
### Description of the Issue
On <= v8.5.2 the ":" in the beginning of the line highlights the whole line.
I installed the new version because of the security fixes v8.5.7 then I noticed that ":" in the beginning of the line only highlights the first word, if there is an space, the highlight ends.

### Steps to Reproduce the Issue
Change the language syntax to "Batch", test text I used in this example:

> ::new new new new 
> :new new new new 
> new new new new 
> :new 11111 5555 new 
> new new new new 
> :new_new-new new

**v8.5.2 and before:**
![image](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/58880202/b87ad608-57b3-4166-b8c9-6b22890cdd5e)

**v8.5.7+:**
![image](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/58880202/76d1cf66-458a-4689-a241-46614c7610d3)

### Expected Behavior
I understand that probably the regex syntax was changed on purpose for language "batch", perhaps a mistake, I don't know for sure.
I expected the highlight to be the same as it was on version v8.5.2 backwards

### Actual Behavior
Highlight for "batch" language is different on version v8.5.2 backwards VS v8.5.7 onwards
I didn't tested the version in between v8.5.2:v8.5.7, therefore, I don't know what is the state of these version.

### Debug Information
N/A

### Additional Info
I did search for "batch" and "highlight" and I didn't find anything, so if this is duplicated, I'm sorry for it.

**Comment by alankilborn** [comment]
Wasn't this changed in Scintilla?
Was it a bug fix or was the behavior left configurable?
I suppose these are rhetorical questions as I could go look it up myself. :-)

**Comment by mpheath** [comment]
The after label style (SCE_BAT_AFTER_LABEL) was added to the Batch lexer intentionally at Lexilla as someone requested it to style like a comment or as another style. It was released in Notepad++ v8.5.4. This forum [comment](https://community.notepad-plus-plus.org/post/87349) displays how to add the new style into an xml file to customize.

Github also shows similar ***label*** and ***after label*** styling.

```bat
:: Comment_style
:Label_style After_label_style
```
The label is `Label_style` . `After_label_style` is ignored by the CMD interpreter when it searches for the label.

If you want the old look, then for the default theme in *stylers.xml*, look for a line with

```xml
<LexerType name="batch" desc="Batch" ext="">
```

and a few lines below it will be

```xml
<WordsStyle name="LABEL" styleID="3" fgColor="FF0000" bgColor="FFFF80" fontName="" fontStyle="1" fontSize="" />
```

duplicate the line and then change name `LABEL` to `AFTER LABEL` and change styleID `3` to `8` so you get


```xml
<WordsStyle name="LABEL" styleID="3" fgColor="FF0000" bgColor="FFFF80" fontName="" fontStyle="1" fontSize="" />
<WordsStyle name="AFTER LABEL" styleID="8" fgColor="FF0000" bgColor="FFFF80" fontName="" fontStyle="1" fontSize="" />
```

**Comment by kiske666** [comment]
Hello @mpheath  - I was searching for some info on Lexilla to see if I could find anything in any of release notes, but I'm not well experienced in this subject, obviously I didn't find anything.

I just changed the "stylers.xml" and it worked flawlessly! 

Thanks.
